org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/extensions/codeaction/ICodeActionParticipant.java

```diff
@@ -33,7 +34,9 @@ public interface ICodeActionParticipant {
 	 * @param codeActions   list of code actions to fill.
 	 * @param cancelChecker the cancel checker.
 	 */
-	void doCodeAction(ICodeActionRequest request, List<CodeAction> codeActions, CancelChecker cancelChecker);
+	default void doCodeAction(ICodeActionRequest request, List<CodeAction> codeActions, CancelChecker cancelChecker)

```

Comment:
Could you add in javdoc the trows of cancellableexception by explaining when it can be occur please

Reply:
Done